spotbugs-exclude.xml

```diff
@@ -44,6 +44,12 @@
         <Bug pattern="PATH_TRAVERSAL_OUT" />
     </Match>
     <!--We do intend to take user input.-->
+    <Match>
+        <Class name="com.ocient.cli.extract.RecordExtractorFactory" />
+        <Method name="create" />
+        <Bug pattern="PATH_TRAVERSAL_OUT" />
+    </Match>    
+    <!--We do intend to take user input.-->
```

Comment:
This is spotbug complaining about us potentially taking user input about where to put a file. But thats precisely what we want to do.